src/algebra/lie/basic.lean

```diff
@@ -572,6 +574,10 @@ instance lie_subalgebra_lie_algebra (L' : lie_subalgebra R L) : lie_algebra R L'
 
 namespace lie_subalgebra
 
+variables {R L}
+
+@[simp] lemma zero_mem {L' : lie_subalgebra R L} : (0 : L) ∈ L' := (L' : submodule R L).zero_mem
```

Comment:
Is it worth adding `add_mem` and `smul_mem` while you're here?

Reply:
Done!